web/packages/teleterm/src/ui/ThemeProvider/theme.ts

```diff
@@ -31,12 +31,12 @@ const colors = {
     surface: '#222C59',
     surfaceSecondary: '#182047',
 
-    elevated: '#2D3761',
+    elevated: '#344179',
 
     popout: '#4A5688',
   },
 
-  brand: '#512FC9',
+  brand: '#9F85FF',
```

Comment:
Shouldn't we import those directly from the dark theme instead of copying them here? In the end, shouldn't the Web UI and Connect use the same themes? cc @rudream